typescript/sdk/src/warp/WarpCore.ts

```diff
@@ -491,11 +491,28 @@ export class WarpCore {
     const adapter = destinationToken.getAdapter(this.multiProvider);
     if (
       destinationToken.standard === TokenStandard.EvmHypXERC20 ||
-      destinationToken.standard === TokenStandard.EvmHypXERC20Lockbox
+      destinationToken.standard === TokenStandard.EvmHypXERC20Lockbox ||
+      destinationToken.standard === TokenStandard.EvmHypVSXERC20 ||
+      destinationToken.standard === TokenStandard.EvmHypVSXERC20Lockbox
     ) {
-      destinationBalance = await (
-        adapter as IHypXERC20Adapter<unknown>
-      ).getMintLimit();
+      const adapter = destinationToken.getAdapter(
+        this.multiProvider,
+      ) as IHypXERC20Adapter<unknown>;
+      destinationBalance = await adapter.getMintLimit();
+
+      if (
+        destinationToken.standard === TokenStandard.EvmHypVSXERC20 ||
```

Comment:
How will this standard get set? Right now, it is not yeah? wouldn't we need a corresponding change in the warp core config artifact?

Reply:
yes, the config artifact is only in the UI afaik?

Reply:
this doesnt need to be urgently merged as @Xaroz already made the change on the UI
this just protects against onchain limit changes

Reply:
I have asked @Xaroz to make this change and test with the UI.